From private repo

v-alje opened the pull request.

azurecla commented:
Hi __@brjohnstmsft__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (brjohnst). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, or work for Microsoft Open Technologies, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

AshokKumar-P committed:
[forumsTeam] Minor Typo

Minor Typo Correction

Commit:
Fixed aml_config

bprakash committed:
Update apache-domain-joined-architecture.md

removed on-premises AD

MGoedtel committed:
updated agent support topic

ShawnJackson committed:
edit pass: tutorial-create-azure-ssis-runtime-portal

AlanSt committed:
removing preview

ShawnJackson committed:
edit pass: tutorial-create-azure-ssis-runtime-portal

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-create-azure-ssis-runtime-portal

ShawnJackson committed:
edit pass: tutorial-create-azure-ssis-runtime-portal

ShawnJackson committed:
edit pass: tutorial-create-azure-ssis-runtime-portal

ShawnJackson committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr into tutorial-create-azure-ssis-runtime-portal

ShawnJackson committed:
edit pass: tutorial-create-azure-ssis-runtime-portal

AlanSt committed:
and 'important' section at top

this is a warning to customers that turning on the feature may break things

v-nagta committed:
USER STORY 1171618 SaaS App Tutorial: Confluence SAML SSO by Microsoft Update

mithun-prasad committed:
Update scenario-sentiment-analysis-deep-learning.md

mithun-prasad committed:
updated ms date

v-viinde committed:
USER STORY 1167250 SaaS App Tutorial: lr Tutorial

v-viinde committed:
USER STORY 1172540 SaaS App Tutorial: EBSCO Update

v-viinde committed:
Merge branch 'master' into ebscoupdate

v-viinde committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
Updated

v-viinde committed:
Updated

Brenduns committed:
Updating topic Author in metadata

Brenduns committed:
Deleting topic and media folder for topic. Content is no longer supported and should not be found or used by customers.

Brenduns committed:
Redirect update for deprecated and deleted topic

Brenduns committed:
udpating PowerShell cmdlet to match current form

CawaMS committed:
adding help alias

v-viinde committed:
Updated

v-nagta committed:
updated

v-viinde committed:
Updated

v-nagta committed:
USER STORY 1173993 SaaS App Tutorial: Veritas Enterprise Vault.cloud SSO Update

v-viinde committed:
USER STORY 1173991 SaaS App Tutorial: PatentSQUARE Update

v-viinde committed:
Merge branch 'master' into patentsquareupdate

cephalin committed:
add link to azure dns

cephalin committed:
validation

cephalin committed:
validation

spelluru committed:
PowerShell -> .NET

spelluru committed:
PowerShell -> .NET

Commit:
freshness

giladmit committed:
added screenshot

giladmit committed:
Merge branch 'master' of https://github.com/Microsoft/azure-docs-pr into GiladWorkingBranch

Commit:
acrolinx

Brenduns committed:
Update azure-stack-download-azure-marketplace-item.md

Brenduns committed:
Merge branch 'master' into topic-deprecation

spelluru committed:
minor changes to landing page

cephalin committed:
minor fix

spelluru committed:
Azure Portal section

cephalin committed:
staged review

spelluru committed:
stop IR

spelluru committed:
minor change

ChiragPavecha committed:
Fixing link for Remove-ServiceFabricNodeState cmd

Fixing link for "Remove-ServiceFabricNodeState cmd" under "Behaviors you may observe in Service Fabric Explorer" section

CawaMS committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

anirudhcavale committed:
Updating Supported Metrics

ShawnJackson committed:
Merge branch 'master' into tutorial-create-azure-ssis-runtime-portal

ShawnJackson committed:
Update tutorial-create-azure-ssis-runtime-portal.md

cephalin committed:
staged review

v-geberr committed:
Pattern feature

wesmc7777 committed:
Update image for GA of geo-replication

wesmc7777 committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs-pr

Brenduns committed:
fixing 1711 relnotes link

carolinacmoravia committed:
CC47663: Update network-watcher-packet-capture-manage-portal.md #430

Hello, @jimdial,

This proposed file change comes from this Community Contribution: https://github.com/MicrosoftDocs/azure-docs.ja-jp/pull/430/files

Please, help to check this proposed file change into the article and help to merge if you agree with fix. If not, please, let me know either if you would like me to fix it in another way within this PR, if you prefer to fix it in another PR or if I should close this PR as by-design. In case of using another PR, please, let me know of your PR number, so we can confirm and close this PR.

Many thanks in advance.

Ja-Dunn committed:
Merge pull request #32194 from ShawnJackson/tutorial-create-azure-ssis-runtime-portal

edit pass: tutorial-create-azure-ssis-runtime-portal

v-geberr committed:
fixed anchor

Commit:
automation : image updates #4

sethmanheim committed:
Freshness updates

sethmanheim committed:
More updates

sethmanheim committed:
More updates

cherylmc committed:
update

tamram committed:
fixing encryption section

GitHubber17 committed:
Merge pull request #32563 from giladmit/GiladWorkingBranch

added screenshot

v-geberr committed:
moved regions into reference section

v-shils committed:
Merge pull request #32482 from Brenduns/daily-fixes-0129

udpating PowerShell cmdlet to match current form

Commit:
updated datacenter integration steps

jeffgilb committed:
Merge branch 'master' into jeff131

GitHubber17 committed:
Merge pull request #32471 from Brenduns/topic-deprecation

Topic deprecation

cherylmc committed:
update

GitHubber17 committed:
Merge pull request #32569 from spelluru/adflanding0131

minor changes to landing page

Commit:
Fixed some CLI links.

Commit:
Fixed next steps link

v-geberr committed:
urls

Commit:
updates

cherylmc committed:
update

Commit:
Various fixes

Commit:
merge conflict fix

Commit:
Updates

wesmc7777 committed:
Update image for GA of geo-replication, some additional grammar updates

Commit:
acrolynx fixes

tamram committed:
fix TOC duplication

v-geberr committed:
changes requested

cherylmc committed:
update

Commit:
tutlist14

Commit:
corrected missing sentence

Commit:
Merge pull request #32597 from jimdial/patch-69

Fixed some CLI links.

Commit:
Merge pull request #32438 from v-viinde/ebscoupdate

USER STORY 1172540 SaaS App Tutorial: EBSCO Update

Ja-Dunn committed:
Merge pull request #32581 from wesmc7777/master

Update image for GA of geo-replication

PRMerger10 committed:
Merge pull request #32553 from v-viinde/patentsquareupdate

USER STORY 1173991 SaaS App Tutorial: PatentSQUARE Update

Commit:
Merge pull request #32318 from v-nagta/confluencev1

USER STORY 1171618 SaaS App Tutorial: Confluence SAML SSO by Microsof…

Ja-Dunn committed:
Merge pull request #32591 from cherylmc/troubleshoot

troubleshoot link

PRMerger13 committed:
Merge pull request #32575 from anirudhcavale/patch-2

Updating Supported Metrics

PRMerger12 committed:
Merge pull request #32584 from sethmanheim/fresh-sb

Freshness updates

Ja-Dunn committed:
Merge pull request #32607 from MarkusVi/tutlist14

tutlist14

Lisaco88 committed:
Merge pull request #32561 from spelluru/tocfix0131

PowerShell -> .NET

Ja-Dunn committed:
Merge pull request #32599 from georgewallace/imageupdates

automation : image updates #4

Ja-Dunn committed:
Merge pull request #32596 from v-geberr/0131-reference-regions

moved regions into reference section

Lisaco88 committed:
Merge pull request #32572 from spelluru/vnetchanges0130

Azure Portal section

PRMerger14 committed:
Merge pull request #32120 from AlanSt/patch-4

removing preview

Commit:
Fixed header # instead of ##!

Lisaco88 committed:
Fixed single-sign on typo

PRMerger19 committed:
Merge pull request #32588 from tamram/tamram-0129

fixing encryption section

Lisaco88 committed:
Merge pull request #32552 from v-nagta/veritas

USER STORY 1173993 SaaS App Tutorial: Veritas Enterprise Vault.cloud …

v-alje committed:
Merge branch 'master' of https://github.com/MicrosoftDocs/azure-docs into FromPublicRepo

PRMerger14 committed:
Merge pull request #32582 from Brenduns/daily-0131

fixing 1711 relnotes link

Lisaco88 committed:
Merge pull request #32437 from v-viinde/lr

USER STORY 1167250 SaaS App Tutorial: lr Tutorial

Lisaco88 committed:
Merge pull request #32604 from tamram/tamram-0131

fix TOC duplication

Lisaco88 committed:
Merge pull request #32595 from jeffgilb/jeff131

updated datacenter integration steps

v-alje committed:
Merge pull request #32618 from MicrosoftDocs/FromPublicRepo

From public repo

PRMerger12 committed:
Merge pull request #32580 from v-geberr/0131-faq

Pattern feature

PRMerger10 committed:
Merge pull request #32583 from carolinacmoravia/patch-590

CC47663: Update network-watcher-packet-capture-manage-portal.md #430

PRMerger18 committed:
Merge pull request #32573 from ChiragPavecha/patch-4

Fixing link for Remove-ServiceFabricNodeState cmd

PRMerger18 committed:
Merge pull request #32570 from cephalin/container

minor fix

Commit:
Merge pull request #32562 from Blackmist/fresh18

freshness

PRMerger14 committed:
Merge pull request #32538 from CawaMS/master

adding help alias for Profiler and Debugger

PRMerger15 committed:
Merge pull request #32402 from mithun-prasad/patch-9

Update scenario-sentiment-analysis-deep-learning.md

PRMerger18 committed:
Merge pull request #32409 from mithun-prasad/patch-10

updated ms date

Commit:
Merge pull request #32030 from MGoedtel/UpdateLATopics1242018

updated agent support topic

Commit:
Merge pull request #31327 from raymondlaghaeian/patch-21

Fixed aml_config

PRMerger17 committed:
Merge pull request #31936 from bprakash/patch-19

Update apache-domain-joined-architecture.md

PRMerger19 committed:
Merge pull request #30848 from AshokKumar-P/patch-13

[forumsTeam] Minor Typo

AnithaAdusumilli committed:
Vnet service endpoints general availability

Ja-Dunn committed:
Merge pull request #32560 from cephalin/dns

add link to azure dns

Commit:
Merge pull request #32622 from AnithaAdusumilli/patch-6

Vnet service endpoints general availability
